pkgs/kde/third-party/krohnkite/default.nix

```diff
@@ -0,0 +1,34 @@
+{
+  lib,
+  stdenv,
+  fetchurl,
+  kwin,
+  kpackage,
+}:
+stdenv.mkDerivation (finalAttrs: {
+  pname = "krohnkite";
+  version = "0.9.7";
+
+  src = fetchurl {
+    url = "https://github.com/Ben9986/krohnkite/releases/download/${finalAttrs.version}/krohnkite-${finalAttrs.version}.kwinscript";
```

Comment:
Any reason to not fetchFromGitHub here? And why your own fork?

Reply:
git shenanigans sorry, I used the wrong stash. Will change to upstream. The `.kwinscript` file isn't in the source tree but is in the releases. Is there a better way to fetch a  file from a release tag?

Reply:
Can we build from source instead?

Reply:
yeh absolutely. Its an NPM project so i thought the script would be preferable.